libs/core/src/ports/interfaces.ts

```diff
@@ -306,13 +306,6 @@ export type NotificationsProviderRecipient =
 
 type BaseNotifProviderOptions = {
   users: { id: string; email?: string }[];
-  actor?: {
```

Comment:
The actor is a `Knock` specific term. The actor _should_ be passed when triggering the Knock workflow. If we remove it from the options here then the actor in the `KnockProvider` should be derived from the data payload. See https://github.com/hicommonwealth/commonwealth/blob/master/libs/adapters/src/knock/index.ts#L109

Reply:
It's not really the user who triggers the workflow but rather the user who causes it to be triggered. Either nbd to not include it now just nice to have in long run by default since it facilitates building workflows + traceability on Knock side.